src/gulpfile.js

```diff
@@ -4,6 +4,8 @@ const debug = require('debug')('slate-tools');
 const argv = require('yargs').argv;
 const runSequence = require('run-sequence');
 
```

Comment:
Thought we were separating third party modules from modules of our own - e.g. https://github.com/Shopify/slate-tools/blob/master/src/tasks/build-css.js#L1-L9